task08-multiple-languages-optional/methods.js

```diff
@@ -0,0 +1,71 @@
+
+/*Order the list by alphabetical order*/ 
+function sortJSON(data, key) {
+  return data.sort(function (firstElement, secondElement) {
+      var firstCompany = firstElement[key]
+      var secondCompany = secondElement[key];
+      if (firstCompany < secondCompany){
+        return -1;
+      }
+      else if (firstCompany > secondCompany){
+        return 1;
+      }
+      else{
+        return 0;
+      }
+  });
+}
+
+/*Translate the data*/
+const translateData = async (companies) => {
+  for (let index = 0; index < 7; index++) {
+    let item = companies[index].description;
+    const spanish = await translate(item, { to: "es" });
+    companies[index].description = spanish;
+  }
+  return companies;
+};
+
+/*Read the JSON and create a list of objects*/
+let getCompanies = new Promise(async function (resolve, reject) {

```

Comment:
No need to wrap async function in a `Promise` it's already a promise. Use the `return` instead of `resolve` and `throw Error("Error msg);` instead of `reject`